tests/projects/c++/modules/link_order/xmake.lua

```diff
@@ -3,13 +3,15 @@ set_languages("c++20")
 
 target("foo")
     add_rules("c++")
-    set_kind("static")
-    add_files("src/foo.mpp", {public = true})
```

Comment:
why not use `static` and `public` now?

Reply:
because a static library with only named module doesn't make sense, module objectfiles are builded by executable so it'll only result on a empty .a / .lib

Reply:
> so it is only a module objectfiles set? Is it possible to use object kind?

no objectfile is built, only the deps scanning step will be run (for module installation)
it behave exactly like a headeronly library (but tied to a runtime and with deps scanning)

Reply:
got it, or you can try to add `moduleonly` kind.